config/vendor-api-v1.yml

```diff
@@ -1175,6 +1175,16 @@ components:
           - "80"
           - "90"
           - "98"
+        other_disability_details:
+          type: string
+          nullable: true
+          description: The candidate’s description of their disability, if they selected “Other”. Corresponds to the HESA disability code `96`
+          example: "Partially sighted"
+        other_ethnicity_details:
+          type: string
+          nullable: true
+          description: The candidate’s description of their ethnicity, if they selected “Other”. Corresponds to the HESA ethnicity code `80`
+          example: "Hawaiian"
```

Comment:
I've guessed at these examples with what I think is sensible. Do we have an examples of the intended use of this field?

Reply:
Maybe we keep it ambiguous and have something like "My disability is ..." or something similar, same with ethnicity